packages/cli-tool/src/helpers/hook-error.ts

```diff
@@ -0,0 +1,13 @@
+type Result = {failures: {error: Error}[]};
+
+export const hookFailed = ({failures} :Result):boolean => {
```

Comment:
Maybe add some spaces here as well, and on the next function in this file as well.
```suggestion
export const hookFailed = ({failures}: Result): boolean => {
```

Reply:
All spacing issues fixed in: 7269e17 ;-)